README.md

```diff
@@ -63,6 +64,62 @@ python setup.py test
 sudo python setup.py install
 ```
 
+Getting Started (Mac OS X)
+-------------------------
+**Getting the source code**
+
+Get the source code from GitHub using :
+
+`git clone https://github.com/G-Node/nixpy.git`
+
+**Build NIXPY under OS X**
+
+_Dependecies_
+
+- NIXPY requires installation of [NIX](https://github.com/G-Node/nix). Make sure you install NIX first.
+- boost-python.
+ Install it using homebrew. `brew install boost-python`
+- Python (version = 2.7.x) + python-setuptools + python-numpy (version >= 1.8.0). 
+
+OS X default python works well.
+
+_Instruction to build_
+
+```
+bash
+
+# install dependencies if required
+sudo apt-get install libboost-python-dev build-essential python-dev python-setuptools python-numpy
+
+# build
+python setup.py build
+
+# unit tests
+python setup.py test
+
+# install 
+sudo python setup.py install
```

Comment:
- [Homebrew]  installed python would be fine without sudo.
- [Macports] I wouldn't suggest

We could suggest to install anaconda and then install nixpy. `conda` automatically changes the `.bashrc`.
And, the new version of OS X, has a feature called `rootless`, which doesn't allow critical system files to be modified even with `sudo` .